articles/quickstart/webapp/aspnet-core/_includes/_login.md

```diff
@@ -188,9 +195,13 @@ public class AccountController : Controller
 }
 ```
 
-At this point ASP.NET Core will call `SignOutAsync` for the "Auth0" authentication scheme (i.e. the OIDC authentication handler), but the OIDC middleware does not know what the actual Logout URL is it should call to log the user out of Auth0. To do this you should handle the `OnRedirectToIdentityProviderForSignOut` event when registering the OIDC authentication handler.
+ASP.NET Core calls `SignOutAsync` for the "Auth0" authentication scheme. You need to provide the OIDC middleware with the URL for logging the user out of Auth0. To get the URL, handle the `OnRedirectToIdentityProviderForSignOut` event when you register the OIDC authentication handler.
+
+When the application calls `SignOutAsync` for the OIDC middleware, it also calls the `/v2/logout` endpoint of the Auth0 Authentication API. The user is logged out of Auth0.
```

Comment:
> When the application calls `SignOutAsync` for the OIDC middleware, it also calls the `/v2/logout` endpoint of the Auth0 Authentication API. The user is logged out of Auth0.

Why are you breaking this up into two sentences? This does not read nicely. Please make it 

> When the application calls `SignOutAsync` for the OIDC middleware, it also calls the `/v2/logout` endpoint of the Auth0 Authentication API, logging the user out of Auth0.